lib/node/nodes/georeference-city.js

```diff
@@ -7,7 +7,8 @@ var PARAMS = {
     source: Node.PARAM.NODE(Node.GEOMETRY.ANY),
     city: Node.PARAM.STRING(),
     admin_region: Node.PARAM.STRING(),
-    country: Node.PARAM.STRING()
+    country: Node.PARAM.STRING(),
+    provider: Node.PARAM.NULLABLE(Node.PARAM.STRING(), 'mapzen')
```

Comment:
Makes sense. Will iterate over that and name the parameter to use the generic function (no provider defined in the signature, uses whatever the user has configured in his/her account) as `user_default` or similar. Thanks!